package.json

```diff
@@ -95,6 +96,7 @@
     "city-timezones": "^1.2.1",
     "eslint": "^8.34.0",
     "lucide-react": "^0.171.0",
+    "react-beautiful-dnd": "^13.1.1",
```

Comment:
react-beautiful-dnd is basically on life support, and did't have any new features for around 3 years now, maybe it's worth consider something more modern and maintained